### PR TITLE
Use cocina to get the default object rights for the APO

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?

Migrate away from direct Fedora access.

## Was the usage documentation (e.g. README, DevOpsDocs, consul, wiki, queue specific README) updated?

no

### CODE REVIEW

File: lib/was_crawl_preassembly/content_metadata_generator_service.rb [hunk 1]
```diff
@@ -33,11 +33,12 @@ def do_post_transform(metadata_content)
       end
 
       def template_suffix
-        druid_obj = Dor.find @druid_id
-        admin_policy = druid_obj.admin_policy_object
-        default_rights_md = admin_policy.datastreams['defaultObjectRights'].content
-        # If the read access is anything otherthan world, it's dark.
-        Nokogiri::XML(default_rights_md).xpath('//access[@type="read"]/machine/world').first.nil? ? 'dark' : 'public'
+        object_client = Dor::Services::Client.object(@druid_id)
+        cocina_model = object_client.find
```
Comment (justinlittman) [inline lib/was_crawl_preassembly/content_metadata_generator_service.rb:37]: Minor request: Can we call this a `dro`?

File: lib/was_crawl_preassembly/utilities.rb [hunk 2]
```diff
@@ -1,14 +1,16 @@
 module Dor
   module WASCrawl
     class Utilities
-      def self.get_collection_id(druid_obj)
-        collection = druid_obj.collections.first
-        collection.id.sub('druid:', '')
-      rescue => e
-        raise "#{druid_obj.id} doesn't belong to a collection (#{e.message})"
+      # @param [Cocina::Models::DRO] cocina_model
```
Comment (justinlittman) [inline lib/was_crawl_preassembly/utilities.rb:4]: Same request with calling this `dro`.

File: lib/was_crawl_preassembly/utilities.rb [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+      def self.get_collection_id(cocina_model)
+        collection = cocina_model.structural.isMemberOf
+        raise "#{cocina_model.externalIdentifier} doesn't belong to a collection" unless collection
+
+        collection.delete_prefix('druid:')
       end
 
-      def self.get_crawl_id(druid_obj)
+      # @param [Cocina::Models::DRO] cocina_model
+      def self.get_crawl_id(_cocina_model)
```
Comment (justinlittman) [inline lib/was_crawl_preassembly/utilities.rb:13]: Where is `druid_obj` coming from?

File: robots/wasSeedDissemination/update_thumbnail_generator.rb [hunk 4]
```diff
@@ -18,8 +18,7 @@ def initialize
         def perform(druid)
           druid_obj = Dor.find(druid)
```
Comment (justinlittman) [inline robots/wasSeedDissemination/update_thumbnail_generator.rb:19]: Still using Dor here?

Comment (jcoyne) [reply robots/wasSeedDissemination/update_thumbnail_generator.rb:19]: yes, on line 20.

File: lib/was_crawl_preassembly/content_metadata_generator_service.rb [hunk 5]
```diff
@@ -33,11 +33,12 @@ def do_post_transform(metadata_content)
       end
 
       def template_suffix
-        druid_obj = Dor.find @druid_id
-        admin_policy = druid_obj.admin_policy_object
-        default_rights_md = admin_policy.datastreams['defaultObjectRights'].content
-        # If the read access is anything otherthan world, it's dark.
-        Nokogiri::XML(default_rights_md).xpath('//access[@type="read"]/machine/world').first.nil? ? 'dark' : 'public'
+        object_client = Dor::Services::Client.object(@druid_id)
+        dro = object_client.find
```
Comment (justinlittman) [inline lib/was_crawl_preassembly/content_metadata_generator_service.rb:37]: L36-37 could be combined.

**Pull request merged**